sakuraio/software/apis/project.py

```diff
@@ -0,0 +1,49 @@
+from sakuraio.software.exceptions import UnSupportedExceptions
+from sakuraio.software.utils import validate_sort_project, validate_datastore, validate_location
+
+
+class ProjectMixins(object):
+    def show_projects(self, name=None, sort=None):
+        # validate sort option
+        if validate_sort_project(sort) is not False:
+            raise UnSupportedExceptions('sort', sort)
+
+        return self.do('GET', 'projects',
+                       query_params={'name': name, 'sort': sort}
+                       )
+
+    def create_project(self, name, datastore='light', location='false'):
+        if validate_datastore(datastore) is not True:
```

Comment:
This condition is just simple `if not validate_datastore(datastore)` ?

Reply:
I think so too. I changed it on this commit. https://github.com/sakuraio/python-sakuraio/pull/23/commits/1e1f49024e0431d6f9f018a3328941c44d9d64f1